test/testdata/namer/fuzz_repeated_kwarg.rb

```diff
@@ -1,19 +1,19 @@
 # typed: false
 # from fuzzer: https://github.com/sorbet/sorbet/issues/1133
-def f1(x, x); end # error: Duplicated argument name `x`
-def f2(x, x=0); end # error: Duplicated argument name `x`
-def f3(x:, x:); end # error: Duplicated argument name `x`
-def f4(x:, x: nil); end # error: Duplicated argument name `x`
-def f5(x, x:); end # error: Duplicated argument name `x`
-def f6(x, x: nil); end # error: Duplicated argument name `x`
-def f7(x=0, x: nil); end # error: Duplicated argument name `x`
+def f1(x, x); end # error: Parse Error: duplicate argument name
+def f2(x, x=0); end # error: Parse Error: duplicate argument name
+def f3(x:, x:); end # error: Parse Error: duplicate argument name
+def f4(x:, x: nil); end # error: Parse Error: duplicate argument name
+def f5(x, x:); end # error: Parse Error: duplicate argument name
+def f6(x, x: nil); end # error: Parse Error: duplicate argument name
+def f7(x=0, x: nil); end # error: Parse Error: duplicate argument name
```

Comment:
This is a regression in error message quality to me:

- The `Parse Error:` prefix is inconsistent with every other error message from Sorbet. (Sorbet doesn't say what pass the error came from, only what the error is.)
- The error message no longer mentions which argument is duplicated.

Can we fix either or both of these?

Reply:
well, it needs explanation I guess. In master Sorbet's parser doesn't emit any errors for this case, it parses it into an AST and so later some step generates an error for it. So now there are two errors:
```
$ SORBET_SILENCE_DEV_MESSAGE=1 bazel-bin/main/sorbet -p parse-tree-whitequark --no-error-count -e 'def m(a, a); end'
s(:def, :m,
  s(:args,
    s(:arg, :a),
    s(:arg, :a)), nil)
-e:1: Parse Error: duplicate argument name https://srb.help/2001
     1 |def m(a, a); end
                 ^

-e:1: Duplicated argument name a https://srb.help/4018
     1 |def m(a, a); end
                 ^
    -e:1: In argument list of method
     1 |def m(a, a); end
        ^^^^^^^^^^^
```

Technically it's a parsing error, should I remove the code that gives an error after parsing?

> The `Parse Error:` prefix is inconsistent with every other error message from Sorbet. (Sorbet doesn't say what pass the error came from, only what the error is.)
That's a prefix that every parse error has. Should I remove it?

> The error message no longer mentions which argument is duplicated.
Yes, that's what what whitequark/parser returns. Not sure if I can change but I'll give it a try. And I'll try to change an `unexpected token` error as well to include the token name.